src/components/FarmCard/FarmCardDetails.tsx

```diff
@@ -277,126 +296,218 @@ const FarmCardDetails: React.FC<{
                 </Box>
-                <Box color={palette.success.main}>
-                  <Typography variant='body2'>{apyWithFee}%</Typography>
-                </Box>
-              </Box>
-            </>
-          )}
-          {!stakingInfo.ended && (
-            <Box className={classes.buttonWrapper} mt={isMobile ? 2 : 0}>
-              <Box display='flex' justifyContent='space-between'>
-                <Typography variant='body2'>In Wallet:</Typography>
                 <Box
+                  mt={2}
+                  width={1}
                   display='flex'
-                  flexDirection='column'
-                  alignItems='flex-end'
-                  justifyContent='flex-start'
+                  justifyContent='space-between'
                 >
-                  <Typography variant='body2'>
-                    {formatTokenAmount(userLiquidityUnstaked)} LP{' '}
-                    <span>({getUSDString(stakedAmounts?.unStakedUSD)})</span>
-                  </Typography>
-                  <Link
-                    to={`/pools?currency0=${getTokenAddress(
-                      token0,
-                    )}&currency1=${getTokenAddress(token1)}`}
-                    style={{ color: palette.primary.main }}
+                  <Box display='flex' alignItems='center'>
+                    <Typography variant='body2' color='textSecondary'>
+                      APY
+                    </Typography>
+                    <Box ml={0.5} height={16}>
+                      <img src={CircleInfoIcon} alt={'arrow up'} />
+                    </Box>
+                  </Box>
+                  <Box color={palette.success.main}>
+                    <Typography variant='body2'>{apyWithFee}%</Typography>
+                  </Box>
+                </Box>
+              </>
+            )}
+            {!stakingInfo.ended && (
+              <Box className={classes.buttonWrapper} mt={isMobile ? 2 : 0}>
+                <Box display='flex' justifyContent='space-between'>
+                  <Typography variant='body2'>{t('inwallet')}:</Typography>
+                  <Box
+                    display='flex'
+                    flexDirection='column'
+                    alignItems='flex-end'
+                    justifyContent='flex-start'
                   >
-                    Get {currency0?.symbol} / {currency1?.symbol} LP
-                  </Link>
+                    <Typography variant='body2'>
+                      {formatTokenAmount(userLiquidityUnstaked)} LP{' '}
+                      <span>({getUSDString(stakedAmounts?.unStakedUSD)})</span>
+                    </Typography>
+                    <Link
+                      to={`/pools?currency0=${getTokenAddress(
+                        token0,
+                      )}&currency1=${getTokenAddress(token1)}`}
+                      style={{ color: palette.primary.main }}
+                    >
+                      Get {currency0?.symbol} / {currency1?.symbol} LP
```

Comment:
Templates, please review the rest of the file for other locations where we can use message consts